src/plugins/unified_search/public/actions/apply_filter_action.ts

```diff
@@ -7,11 +7,14 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { ThemeServiceSetup } from '@kbn/core/public';
+import type { ThemeServiceSetup } from '@kbn/core/public';
 import { toMountPoint } from '@kbn/kibana-react-plugin/public';
-import { Action, createAction, IncompatibleActionError } from '@kbn/ui-actions-plugin/public';
-import { Filter, FilterManager, TimefilterContract, esFilters } from '@kbn/data-plugin/public';
-import { getOverlays, getIndexPatterns } from '../services';
+import { createAction, IncompatibleActionError } from '@kbn/ui-actions-plugin/public';
+import type { Action } from '@kbn/ui-actions-plugin/public';
+import { FilterManager, TimefilterContract } from '@kbn/data-plugin/public';
+import { esFilters } from '@kbn/data-plugin/public';
```

Comment:
Why don't we combine it with the above import?

Reply:
Yes, sure we can. I separated them because in the future I was going to replace the deprecated esFilters. And in the  [issue](https://github.com/elastic/kibana/issues/131292#issue-1222080170) I described why I could not solve this issue in this PR